### ISSUE TITLE
Sony XAVC-S 4K  and  XAVC-S HD issues

### ISSUE THREAD

**Issue by andrenalin2000** [mifi/lossless-cut#690]
I am a new here. At first many thanks for this useful cutting tool!
I use it 2 years and it was nearly perfect for cutting files from different cameras. Until I got a Sony ZV-1.

Latest LosslessCut for Windows 64. 8 Cores AMD.
As mentioned here the Sony XAVC-S MP4 is not quite MP4 compliant. But I can cut it to MOV and Matroska. With issues.

Both 4K30 and 1080/30/60/120:
1. Viewing - Playback.
Seek to next keyframe. Then move forward frame by frame  (30 frames in 1 keyframe) and after 28-th frame you will see 30-th frame and then 30-th frame again. And then the next keyframe.

2. Cutting ( keyframe cut and normal cut).
-You lose 1 additional keyframe from the beginning of the cut segment if the Cut Start in the beginning of this keyframe.
-You lose 1 1/2 keyframe from the beginning of the cut segment if the Cut Start in the middle of the previous keyframe.

I tried again another files from Gopro, DJI Osmo - no these issues.

The main issue -  4K30 only:

While cutting you lose 1 keyframe or more from the CutStart point, as mentioned before. BUT ONLY IN VIDEO TRACK.

The Audio track cutting some another way. I dont know how. I tried Keyframe and Normal Cut and different positions of Cut Start and End. Every time in the Cut Segment the Audio is longer then Video. When you playback it you see the dark screen and audio and then the video and audio. 

The sample of the Sony's XAVC-S MP4 4K30  
https://photos.app.goo.gl/Ww1GB5BSXVqmbbw26

Thanks,
Andrei

In order to make it easier for me to help you, I ask that you please try these things first:
- Try with the newest version from GitHub: https://github.com/mifi/lossless-cut/releases/latest
- Check https://github.com/mifi/lossless-cut and **Known issues** / **Troubleshooting**
- Search for your problem under Issues or Discussions
- Please try other type of file and see if it works.

**Share information to help me understand**
- If you have an issue cutting a particular file, please also share the file, if possible. (or mail a link to finstaden@gmail.com if private.) Also please try to cut it with the ffmpeg command line directly.
- If you have a problem or an unexpected bug or crash, please open developer tools after starting LosslessCut and before doing the failing operation. Then share the output from the javascript Console. (Menu: Tools > Toggle Developer Tools)
- Please provide the error report if you got one (Menu: Help -> Report an error)

**Describe the bug**

**To Reproduce**
1. Open file '...'
2. Click on '...'
3. See error

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]

**Additional context**

**Comment by mifi** [comment]
Hi.
I can confirm this issue also with your provided file, but for me it's only an issue with Normal cut. With keyframe cut it cuts correctly. What is your version and your export options?

**Comment by andrenalin2000** [comment]
Hi!

**Comment by andrenalin2000** [comment]
Hi! 
   The version was 3.33.1  Now I am using 3.34.3
And today I CAN export to MOV and to MKV!!!  What I did wrong before??!!
The export options: Keyframe cut; 
MOV faststart? - YES;
Preserve all MP4/MOV metadata? - Yes;
Shift timestamps - make_zero

I tried again the different files from Sony 4K30, 1080/30/60/120 - everything works! 

Seems the only one small Cons - in the exported files the audio is longer then video up to  100-110 ms or around 3 frames.
In the hi frame rate videos the difference is about 20-25 ms (3 frames too).  
This is according to MediaInfo v20.09 Copyright (c) MediaArea.net 

The MPC-HC video player also show the difference. Select File > Properties (Shift-F10)
